src/pages/tasks/NewTaskPage.tsx

```diff
@@ -178,7 +178,7 @@ function NewTaskPage({task, reports, personalDetails}: NewTaskPageProps) {
                                 description={shareDestination?.displayName ? shareDestination.subtitle : translate('newTaskPage.shareSomewhere')}
                                 icon={shareDestination?.icons}
                                 onPress={() => Navigation.navigate(ROUTES.NEW_TASK_SHARE_DESTINATION)}
-                                interactive={!task?.parentReportID}
+                                interactive
```

Comment:
I'm a little bit outdated about NewTaskPage, but can you explain why `interactive` will be always true here?

Reply:
good catch, it was mistakenly pushed, thanks!